Dokumentasjon/V1/ClassDiagrams/no.ks.fiks.saksfaser.v1.felles.skjerming/classdiagram.pu

```diff
@@ -0,0 +1,17 @@
+@startuml
+
+class Skjerming {
+    *tilgangsrestriksjon: Tilgangsrestriksjon
+    *skjermingshjemmel: string
+    skjermingOpphoererDato: date
+    skjermingOpphoererAksjon: date
```

Comment:
Dette er et kodeobjekt. Antagelig trengs det ikke her med mindre vi skal ha det likt som Arkiv. Informasjonen er kun relevant for arkivet.